eclipse-collections-code-generator/src/main/resources/impl/map/immutable/immutablePrimitiveObjectSingletonMap.stg

```diff
@@ -681,6 +682,12 @@ final class Immutable<name>ObjectSingletonMap\<V> extends AbstractImmutable<name
         return Maps.mutable.with(keyFunction.valueOf(this.value1), valueFunction.valueOf(this.value1));
     }
 
+    @Override
+    public \<NK, NV, R extends Map\<NK, NV>\> R toMap(Function\<? super V, ? extends NK> keyFunction, Function\<? super V, ? extends NV> valueFunction, R target)
+    {
+        return target;
```

Comment:
This should either be deleted or call target.put() with the single key and value first.